simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/HeartbeatScheduler.java

```diff
@@ -10,15 +10,14 @@
 @FieldDefaults(makeFinal = true)
 public class HeartbeatScheduler {
 
-    private static final int INITIAL_TASK_DELAY_IN_SECONDS = 1;
     private static final int TASK_DELAY_IN_SECONDS = 1;
 
     private HeartbeatSenderTask heartbeatTask;
 
     public HeartbeatScheduler(StationState stationState, StationMessageSender stationMessageSender) {
         this.heartbeatTask = new HeartbeatSenderTask(stationState, stationMessageSender);
         Executors.newSingleThreadScheduledExecutor().scheduleAtFixedRate(
-                heartbeatTask, INITIAL_TASK_DELAY_IN_SECONDS, TASK_DELAY_IN_SECONDS, TimeUnit.SECONDS);
+                heartbeatTask, TASK_DELAY_IN_SECONDS, TASK_DELAY_IN_SECONDS, TimeUnit.SECONDS);
```

Comment:
if you move this to a constructor, do you still have a NPE? Because this is a background task that ideally should only be run once.